kratos/python_scripts/testing/run_tests.py

```diff
@@ -125,7 +125,7 @@ def RunTestSuit(self, application, applicationPath, path, level, verbose, comman
                         file=sys.stderr)
                     sys.stderr.flush()
 
-    def RunCppTests(self, applications):
+    def RunCppTests(self, applications, verbosity):

```

Comment:
```suggestion
    def RunCppTests(self, applications, verbosity = 1):
```
I would make this optional